src/main/java/com/klarna/hiverunner/HiveRunnerCore.java

```diff
@@ -90,31 +91,22 @@ private HiveShellContainer buildShell(List<? extends Script> scripts, Object tes
   private HiveShellField loadScriptUnderTest(Object testCaseInstance, HiveShellBuilder hiveShellBuilder) {
     try {
       Set<Field> fields = ReflectionUtils.getAllFields(testCaseInstance.getClass(), withAnnotation(HiveSQL.class));
-
       Preconditions.checkState(fields.size() == 1, "Exact one field should to be annotated with @HiveSQL");
-
       Field field = fields.iterator().next();
-      List<Path> scriptPaths = new ArrayList<>();
       HiveSQL annotation = field.getAnnotation(HiveSQL.class);
-      for (String scriptFilePath : annotation.files()) {
-        Path file = Paths.get(Resources.getResource(scriptFilePath).toURI());
-        assertFileExists(file);
-        scriptPaths.add(file);
-      }
-
-      Charset charset = annotation.encoding().equals("") ?
-          Charset.defaultCharset() : Charset.forName(annotation.encoding());
-
+      List<Path> scriptPaths = getScriptPaths(annotation);
+      
       boolean isAutoStart = annotation.autoStart();
-
+      
+      Charset charset = annotation.encoding().equals("") ? Charset.defaultCharset() : Charset.forName(annotation.encoding());
```

Comment:
Move this back to line 105 so it doesn't show up in the diff as it hasn't changed. Let's try get the diff as small as possible with only necessary changes.